neuron.cabal

```diff
@@ -16,6 +16,11 @@ description:
 extra-source-files:
   README.md
   CHANGELOG.md
+data-files:
+  src-js/search.js
+  src-purescript/hello/index.js
```

Comment:
Note that `./src-purescript` is currently unused by the app. (it exists just as a reference implementation to use in future).

Reply:
Oh nevermind. It *is* used as a demonstration. I'll remove it later.